docs/TypeAnnotations.md

```diff
@@ -16,50 +17,7 @@ Mypy is an opt-in type checker. Annotations are optional and, if present, will b
 Say, for example, you're building a converter for ONNX, which converts between ONNX models and the models of some machine learning framework. If you use mypy for your project, it will automatically detect the type hints in the ONNX code and use them to check that you read from and wrote to the ONNX model correctly. Doing this will notify you about identifier typos or wrong data types when accessing attributes of the ONNX protocol buffer or other ONNX features.
 
 ## For ONNX contributors
-If you're contributing to the ONNX repository, you need to add type annotations to your project or CI will fail. Here is how to install it:
-
-#### Python 3
-If you're using Python 3, you can simply install mypy as an extra dependency of onnx.
-
-Call from the top level directory of your onnx repository:
-
-    ~/onnx $ pip install -e .[lint]
-
-And then run the type checks:
-
-    ~/onnx $ python setup.py typecheck
-
-#### Python 2
-The mypy type checker needs python 3 to run (even though it can check python 2 code), so if your onnx development environment is set up with python 2, you need to install mypy into your system python 3 packages (i.e. not into your python 2 virtualenv):
-
-    $ pip3 install mypy
-
-Running then works as it does above for python 3.
-
-    ~/onnx $ python setup.py typecheck
-
-Since you've installed mypy manually, you won't get updates for it when we change the version of the mypy dependency in setup.py, so if you're seeing a CI error and can't reproduce it locally, check your mypy version.
 
 ### What should I annotate?
```

Comment:
This page has one title, one subtitle, one subsubtitle. Maybe too much?

Reply:
Yeah maybe I can removed completely ? I just remove outdated info without touching the rest

Reply:
Maybe remove it completely is not a bad idea.